MathLib/LinAlg/Eigen/EigenLinearSolver.cpp

```diff
@@ -12,6 +12,8 @@
 #include <logog/include/logog.hpp>
 
 #ifdef USE_EIGEN_UNSUPPORTED
+#include <iostream> // to fix compiling errors in Eigen
```

Comment:
The error happens in unsupported/Eigen/src/IterativeSolvers/ConstrainedConjGrad.h, which is included in unsupported/Eigen/IterativeSolvers.h. I checked the history of the file and the header file is not included since 2009. maybe I should change it to include only GMRES.